docker/conf-workers/nginx.conf.j2

```diff
@@ -9,6 +9,22 @@ server {
     listen 8008;
     listen [::]:8008;
 
+    {% if tls_cert_path is not none and tls_key_path is not none %}
+        listen 8448 ssl;
+        listen [::]:8448 ssl;
+
+        ssl_certificate {{ tls_cert_path }};
+        ssl_certificate_key {{ tls_key_path }};
+
+        # Some directives from cipherlist.eu (fka cipherli.st):
+        ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3;
+        ssl_prefer_server_ciphers on;
+        ssl_ciphers "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
+        ssl_ecdh_curve secp384r1; # Requires nginx >= 1.1.0
+        ssl_session_cache shared:SSL:10m;
+        ssl_session_tickets off; # Requires nginx >= 1.5.9
+    {% endif %}
```

Comment:
I assume indentation isn't important here? 😄 

(I usually try to line up the final results, but there's lots of different styles that people use for jinja2 templates...)

Reply:
I mean, you have a point and if this was YAML I certainly couldn't argue with that.

Nginx doesn't care about indentation. I somewhat like indenting the source file because it makes it more readable, though I sympathise that the derived file then looks messy :/.

Reply:
I've never been thrilled with doing it either way. 😢  I'm fine with the way it is!